### ISSUE TITLE
helper method should be completed

### ISSUE THREAD

**Issue by atikul-islam-sajib** [atikul-islam-sajib/FakeIG#2]
1. Call the netG
2. call the netD
3. define the loss function 
4. return Dataloader, models, loss functions

**Comment by atikul-islam-sajib** [comment]
completed
